addons/core/functions/fnc_haveProgrammator.sqf

```diff
@@ -1,21 +1,21 @@
 #include "script_component.hpp"
 
-/*
-    Name: TFAR_fnc_haveProgrammator
-
-    Author(s):
-
-    Description:
-    Returns whether the player has a programmer
-
-    Parameters:
-    Nothing
-
-    Returns:
-    BOOLEAN
-
-    Example:
-    _hasProgrammer = call TFAR_fnc_haveProgrammator;
-*/
-if (isNil {TFAR_currentUnit} || {isNull (TFAR_currentUnit)}) exitWith{false};
-"tf_microdagr" in (assignedItems TFAR_currentUnit);
+/*
+    Name: TFAR_fnc_haveProgrammator
+
+    Author(s):
+
+    Description:
+    Returns whether the player has a programmer
+
+    Parameters:
+    Nothing
+
+    Returns:
+    BOOLEAN
+
+    Example:
+    _hasProgrammer = call TFAR_fnc_haveProgrammator;
+*/
+if (isNil {TFAR_currentUnit} || {isNull (TFAR_currentUnit)}) exitWith{false};
+"tf_microdagr" in (assignedItems TFAR_currentUnit) || {"ACE_microDAGR" in (assignedItems TFAR_currentUnit)};
```

Comment:
It's so we won't have to edit this file if we want to add more items as programmers.